OpenOversight/scripts/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ "${DOCKER_BUILD_ENV:-}" == "production" ]; then
+    gunicorn -w 4 -b 0.0.0.0:3000 app:app
+else
+    yarn build
+    yarn watch &
+    python manage.py runserver
```

Comment:
What's the reason to call `manage.py runserver` here instead of the gunicorn command? Because of [this change](https://github.com/lucyparsons/OpenOversight/pull/523#issuecomment-409296717), running without `DOCKER_BUILD_ENV=production` causes gunicorn to be run on port 5000 rather than 3000.

Reply:
`runserver` is here such that people can get tracebacks and use the Flask in-browser debugger when in the dev env (right now this is not the case which is not ideal)

Reply:
Then would it make sense to add the runserver command back into manage.py, specifying port 3000?